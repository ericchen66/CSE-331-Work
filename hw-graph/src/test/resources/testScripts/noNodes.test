# Test for the nodes of an empty graph

# Creates an empty graph
CreateGraph graph1

# Lists the nodes of the empty graph
ListNodes graph1