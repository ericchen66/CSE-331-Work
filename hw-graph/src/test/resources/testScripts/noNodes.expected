# Test for the nodes of an empty graph

# Creates an empty graph
created graph graph1

# Lists the nodes of the empty graph
graph1 contains: